openwisp_utils/admin_theme/static/admin/css/base_prevent_darkmode.css

```diff
@@ -0,0 +1,52 @@
+/* VARIABLE DEFINITIONS TO PREVENT DARK MODE */
```

Comment:
Instead of creating this new file, just add it in the `openwisp.css` file, please! :smile:

Reply:
Meanwhile, the tests passed, yay!

Reply:
> Instead of creating this new file, just add it in the `openwisp.css` file, please! 😄

Done 👍